package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "eslint",
     "lint:fix": "eslint --fix",
     "test": "npm run test:unit && npm run test:typescript",
-    "test:unit": "tap",
+    "test:unit": "c8 --100 --reporter html node --test",
```

Comment:
Could you remove the tap config?

https://github.com/fastify/fastify-express/blob/a76a18394f3bd22457edc5b39a7255ad7988fcd0/package.json#L88-L90